earth_enterprise/src/fusion/rasterfuse/SConscript

```diff
@@ -31,7 +31,7 @@ gepackgen = env.executable('gepackgen',
                            LIBS=['geautoingest', 'khrasterffio', 'gegdal',
                                  'geraster', 'khffio', 'gefconfigutil',
                                  'geutil', 'gecommon', 'geprotobuf',
-                                 'qt-mt', 'geos'])
+                                 'qt-mt', 'geos', 'gemiscconfig'])
```

Comment:
I don't think this will be necessary if you remove MiscConfig.h from khCache, but I could be wrong.